playgrounds/spring-petclinic-gradle-sb-3/docker/Dockerfile

```
@@ -1,9 +1,27 @@
+WORKDIR /app
+
+COPY --from=training /app/petclinic.jar /app/petclinic.jar
+COPY --from=training /app/app.jsa /app/app.jsa
+
 EXPOSE 8080
 
-CMD ["java", "-jar", "/app/petclinic.jar"]
+ENV JAVA_GC_LOG_OPTS="-Xlog:gc=info,safepoint:/tmp/gc.log::filecount=1,filesize=20M"
+ENV JAVA_APP_CDS_OPTS="-XX:SharedArchiveFile=/app/app.jsa"
+
+ENTRYPOINT exec java \
```

Hadolint (2.14.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)